### PR TITLE
Add JSON linting to eslint

### PR DESCRIPTION
**Pull request by jamesbhobbs**
Add JSON linter to eslint

* [x] Fixes https://github.com/bbc/simorgh/issues/89, Resolves https://github.com/bbc/simorgh/issues/82
* [x] Up-to-date with latest - `git pull --rebase origin latest`
* [x] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)
* [x] Tests pass - `npm test`
* [x] E2E tests pass - `npm run test:e2e` (requires server running see [README](https://github.com/bbc/simorgh#tests) for details)

* [ ] Test engineer approval

### CODE REVIEW

**Comment by jamesbhobbs** [comment]
CodeReviewed Any problem with the insecure dep given that it's only in dev deps? Happy with an alternative suggestion.

**Review by sareh — DISMISSED** [review]

**Review by PaulChinkwende — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "dev": "razzle start",
     "build": "razzle build",
     "test": "npm run test:lint && npm run test:unit",
-    "test:lint": "eslint --ext .js,jsx ./src",
+    "test:lint": "eslint --ext .js,jsx ./src && eslint --ext .json ./data",
```
Comment (jtart) [inline package.json]: Don't need to run the command twice - just add `.json` to `--ext`, and append `./data` after `./src`

**Review by jtart — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (jamesbhobbs) [reply package.json]: @jtart you're right thanks, change made

**Review by sareh — DISMISSED** [review]

**Review by jp-bbc — DISMISSED** [review]

**Review by jamesbhobbs — COMMENTED** [review]
Approved (Jordan's changes)

**Review by jtart — APPROVED** [review]
(Approved Benjamin's changes)

**Comment by jamesbhobbs** [comment]
Approved (Jordan's changes)

**Pull request merged**